lib/instrumentation/index.js

```diff
@@ -70,30 +70,41 @@ function Instrumentation (agent) {
   this._patches = new NamedArray()
 
   for (let mod of MODULES) {
+    if (!Array.isArray(mod)) mod = [mod]
+    const pathName = mod[0]
```

Comment:
Maybe `const pathName = Array.isArray(mod) ? mod[0] : mod`, so you don't need to build an array you don't actually need.